README.md

```diff
@@ -1,38 +1,56 @@
-[Compression results](#nncf-compressed-model-zoo) achievable with the NNCF-powered samples can be found in a table at the end of this document.
+The framework is organized as a Python\* package that can be built and used in a standalone mode. The framework 
+architecture is unified to make it easy to add different compression algorithms for both PyTorch and TensorFlow deep 
+learning frameworks.
+
+The samples demonstrate the usage of compression algorithms for three different use cases on public PyTorch and 
+TensorFlow models and datasets: Image Classification, Object Detection and Semantic Segmentation. 
+[Compression results](#nncf-compressed-model-zoo) achievable with the NNCF-powered samples can be found in a table at 
+the end of this document.
 
 ## Key Features
 
-- Support of various compression algorithms, applied during a model fine-tuning process to achieve best compression parameters and accuracy:
-    - [Quantization](./docs/compression_algorithms/Quantization.md)
-    - [Binarization](./docs/compression_algorithms/Binarization.md)
-    - [Sparsity](./docs/compression_algorithms/Sparsity.md)
-    - [Filter pruning](./docs/compression_algorithms/Pruning.md)
-- Automatic, configurable model graph transformation to obtain the compressed model. The source model is wrapped by the custom class and additional compression-specific layers are inserted in the graph.
-- Common interface for compression methods
-- GPU-accelerated layers for faster compressed model fine-tuning
-- Distributed training support
+- Support of various compression algorithms, applied during a model fine-tuning process to achieve best compression 
+  parameters and accuracy:
+  
+  |Compression algorithm|PyTorch|TensorFlow|
+  | :--- | :---: | :---: |
+  |[Quantization](./docs/compression_algorithms/Quantization.md) | Supported | Supported |
+  |[Mixed-Precision Quantization](./docs/compression_algorithms/Quantization.md#mixed_precision_quantization) | Supported | Support planned |
+  |[Binarization](./docs/compression_algorithms/Binarization.md) | Supported | Not supported |
+  |[Sparsity](./docs/compression_algorithms/Sparsity.md) | Supported | Supported |
+  |[Filter pruning](./docs/compression_algorithms/Pruning.md) | Supported | Supported |
+
+- Automatic, configurable model graph transformation to obtain the compressed model.
+  > **NOTE**: Limited support for TensorFlow models. The models created using Sequential or Keras Functional API are only supported.
+- Common interface for compression methods.
+- GPU-accelerated layers for faster compressed model fine-tuning.
+- Distributed training support.
 - Configuration file examples for each supported compression algorithm.
 - Git patches for prominent third-party repositories ([mmdetection](https://github.com/open-mmlab/mmdetection), [huggingface-transformers](https://github.com/huggingface/transformers)) demonstrating the process of integrating NNCF into custom training pipelines
```

Comment:
Is mmdetection patch still available? I believe the collective decision was to remove it and refer for an OTE fork of mmdetection.

Reply:
@vshampor will we delete mmdetection here or will you make a separate PR?

Reply:
You may delete it here, @aleksei-kashapov was supposed to make a PR, but did not have the time for it, it seems.

Reply:
Done